testing/test_config.py

```diff
@@ -519,7 +519,7 @@ def test_hello(fix):
         """)
         result = testdir.runpytest()
         assert result.parseoutcomes()["pytest-warnings"] > 0
-        assert "hello" not in result.stdout.str()
+        assert "hello" in result.stdout.str()
```

Comment:
I think this code needs to revert the change in the assertion and pass `--disable-pytest-warnings` to `testdir.runpytest` in line 520.